WIP: Sprint 13

dimaspirit opened the pull request:
**Made/Proposed changes:**
- Added fetch API instead $.ajax() and request() (REST API support for NativeScript);
- Chat adapted for NativeScript enviroments;
- Fix TypeError: Cannot read property 'statusCode' of undefined for Node env;
- Add stream management test cases;
- The MessageProxy and the DialogProxy were remove from qbChat.js to separate modules;
- Add link on API ref. in Readme.md;

dimaspirit committed:
add info about API ref in README.md

dimaspirit committed:
Fix TypeError: Cannot read property 'statusCode' of undefined for Node env.

dimaspirit committed:
add stream management tests

grajagandev commented:
Would you be interested in software that automatically finds TypeErrors like this? I am building [Fuzz Stati0n](https://fuzzstati0n.com/?id=284) to do that (free for OSS) - please take a look and consider signing up for our newsletter to keep in touch.

Vladlukhanin committed:
NativeScript support

Added fetch API instead $.ajax() and request() (REST API support for NativeScript);
The MessageProxy and the DialogProxy were remove from qbChat.js to separate modules.
Chat Module adapted for NativeScript
Note:

In Jasmine tests added 2 pending specs (qbAddressbook):
'Get all existed contacts';
'Get contacts in specific address book / with UDID'.

dimaspirit committed:
before release

dimaspirit committed:
before release

dimaspirit committed:
generate new inline docs

dimaspirit committed:
upd Readme
